packages/kolibri-tools/lib/i18n/crowdin/utils.py

```diff
@@ -255,55 +255,58 @@ def _get_plugins_from_list(plugins_list):
         )
 
 
-def _get_plugins_from_locale_paths(locale_paths):
+def _get_locale_paths_from_paths(paths):
     """
-    A helper to handle trying to get the plugins loaded from a given comma-separated list of relative paths
+    A helper to handle trying to get the proper paths loaded from a given comma-separated list of relative paths
     """
-    plugins = []
-    for locale_path in locale_paths.split(","):
+    locale_paths = []
+    for locale_path in paths.split(","):
         if os.path.exists(locale_path):
             locale_data_folder = os.path.join(
                 locale_path, "locale", "en", "LC_MESSAGES"
             )
             logging.info("Found locale data folder: {}".format(locale_data_folder))
-            plugins.append({"locale_data_folder": locale_data_folder})
+            locale_paths.append({"locale_data_folder": locale_data_folder})
         else:
             logging.critical(
                 "{} is not a path that exists relative to here: {}".format(
                     locale_path, os.getcwd()
                 )
             )
             sys.exit(1)
-    return plugins
+    return locale_paths
 
 
-def get_plugins(opts={}):
+def get_locale_paths(opts={}):
     """
     Given the value for a plugin_file or the comma-separated path for any module(s) which we need
-    to interact with, this will return the plugins values as returned by ..webpack_json.initialize_plugins
+    to interact with, this will return the locale_paths values as returned by ..webpack_json.initialize_plugins
+
+    Alternatively, we can use given "locale paths" to get what we need.
 
     Note that per the crowdin.cli documentation for the plugin* args, plugins_path will be attempted
     first if both options are given.
+
+    Returns a set() of paths where locale files can be found
     """
     plugins_path = opts.get("plugins_path")
     plugins_list = opts.get("plugins_list", "")
     locale_paths = opts.get("locale_paths", "")
 
     if plugins_path:
-        plugins = _get_plugins_from_file(plugins_path)
+        paths = _get_locale_paths_from_file(plugins_path)
     elif plugins_list:
-        plugins = _get_plugins_from_list(plugins_list)
+        paths = _get_locale_paths_from_list(plugins_list)
     else:
-        plugins = _get_plugins_from_locale_paths(locale_paths)
-    return plugins
+        paths = _get_locale_paths_from_paths(locale_paths)
+    locale_data_folders = [path["locale_data_folder"] for path in paths]
```

Comment:
Can also use a set comprehension here:
```suggestion
    locale_data_folders = {path["locale_data_folder"] for path in paths}
```